common/autoconf/generated-configure.sh

```diff
@@ -1014,7 +1014,6 @@ infodir
 docdir
 oldincludedir
 includedir
-runstatedir
```

Comment:
Can you regenerate on a newer system so we don't get these unrelated changes?

Reply:
Done